locations/fblipke.yml

```diff
@@ -0,0 +1,152 @@
+---
+location: fblipke
+location_nice: Marzahn - Blumberger Damm 305
+latitude: 52.557725
+longitude: 13.57437
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+  - '@lipkegu:matrix.org'
+hosts:
+# Proxmox VM (QEMU)
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86-64"
+    host__packages__to_merge:
+      - "nano qemu-ga kmod-brcmfmac kmod-brcmutil brcmfmac-firmware-4366c0-pcie"
+    wireless_profile: fblipke
+    wifi_roaming: true
+
+#  NSM5
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m5_xw"
+    wireless_profile: fblipke
+    wifi_roaming: true
+
+# NETWRORKS:
+# Assigned: 10.248.12.0/24; 2001:bf7:850:1b00::/56
+
+# DHCP: 10.248.12.128/25
+# MGMT: 10.248.12.0/26
+# MESH: 10.248.12.64/26
+
+ipv6_prefix: "2001:bf7:850:1b00::/56"
+
+dhcp_no_ping: false
+
+networks:
```

Comment:
I'm aware that some of the commenly used VLAN ids conflict with your home network, however I still want to point out that we usually use the following as listed in (https://github.com/freifunk-berlin/bbb-configs/blob/main/DEVELOPER.md)

10+ for airmax & co mesh links
20+ for 11s mesh
40+ DHCP Clients
42  MGMT
50  Wireguardtunnel